pulsar-broker/src/test/java/org/apache/pulsar/stats/client/PulsarBrokerStatsClientTest.java

```diff
@@ -164,32 +166,47 @@ public void testGetPartitionedTopicMetaData() throws Exception {
         log.info("-- Exiting {} test --", methodName);
     }
 
-    @Test (timeOut = 4000)
-    public void testGetPartitionedTopicDataTimeout() {
+    @Test
```

Comment:
could we create a new test case, the original one is to test timeout, while your new test is try to test retry and success. also  how about add some comments or change the test case name to make it more clear? e.g. according to the test steps, the name like 'testMultiHostUrlRetrySuccess' would be better.  And some of the test cases are not related with `PulsarBrokerStatsClientTest`, It would be better to move them to or create a more related test suite, such as `PulsarMultiHostClientTest`?

Reply:
nice advice, have fixed it.